index.js

```diff
@@ -166,10 +198,9 @@ async function main() {
         new winston.transports.File({
             filename: `./trajectories/${runId}/combined.log`,
             format: winston.format.combine(
-                removeColorsFormat(),
                 winston.format.printf(
                     ({ timestamp, level, message }) =>
-                        `${timestamp} [${level.toUpperCase()}] - ${message}`,
+                        `${timestamp} [${level.toUpperCase()}] - ${stripAnsi(message)}`,
```

Comment:
The ansi stripping didn't need to be so complicated and there were typeissues with the previous way the object was being handled for some reason